Makefile

```diff
@@ -30,4 +30,4 @@ vendor:
 	@echo "✓ Filling vendor folder with library code ..."
 	@go mod vendor
 
-.PHONY: build vendor coverage test lint fmt
\ No newline at end of file
+.PHONY: build vendor coverage test lint fmt
```

Comment:
```suggestion
.PHONY: build vendor coverage test lint fmt

```

Reply:
Committed the suggestion but @pietern feel free to revert all changes to this file